node/node_handler.go

```diff
@@ -201,14 +201,23 @@ func (node *Node) messageHandler(content []byte, sender string) {
 	}
 }
 
-func (node *Node) addStakingTxnIntoPendingTxns(msgPayload []byte) {
+func (node *Node) processStakingMessage(msgPayload []byte) {
 	msg := &message.Message{}
 	err := pb.Unmarshal(msgPayload, msg)
 	if err == nil {
 		stakingRequest := msg.GetStaking()
 		txs := types.Transactions{}
 		if err = rlp.DecodeBytes(stakingRequest.Transaction, &txs); err == nil {
-			utils.GetLogInstance().Error("Successfully added staking transaction to pending list.")
+			for _, tx := range txs {
```

Comment:
we prepared sending money in the genesis https://github.com/harmony-one/harmony/blob/master/node/contract.go#L134

i forgot that yesterday. probably let me change the logic of taking account in harmony from one of the fake accounts which got tokens minted in genesis (constants.go)

Reply:
Don't do it. I've already making the flow working.

Reply:
but eventually we won't do it. it may be working now but eventually for the test-net and devnet we will have to do that way again. We already have the staking preparation here https://github.com/harmony-one/harmony/blob/master/node/contract.go#L137

Reply:
your code won't work for testnet - we will have to switch back to this way again

Reply:
I wrote this PR for you https://github.com/harmony-one/harmony/pull/514/

can you please enable this logic back and test with this PR?

Reply:
Your way is basically batch calling faucet while mine is dynamically calling faucet. There is no major pros or cons on both approach. Mine is just easier for testing and development. And I don't think your approach is significantly better than mine for testnet.